reunioes/progressos/intermediario/php_oo/metodos/Carro.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+    class Carro {
+
+        private $modelo;
+        private $motor;
+        private $ano;
+        
+        public function __construct($a, $b, $c){
+            $this->modelo = $a;
+            $this->motor = $b;
+            $this->ano = $c;
+        }
+
+        public function __toString(){
+            return $this->modelo . ", " . $this->motor . ", " . $this->ano;
```

Comment:
Sempre opte por aspas simples, quando for utilizar puramente strings. Faz com que o PHP não tente processar a informação dentro das aspas, exemplo:

```php
return $this->modelo . ', ' . $this->motor . ', ' . $this->ano;
```